configs/sites/frontera/packages.yaml

```diff
@@ -0,0 +1,256 @@
+packages:
+  all:
+    compiler:: [intel@23.1.0]
+    #compiler:: [gcc@12.2.0]
+    providers:
+      mpi:: [intel-oneapi-mpi@2021.9.0]
+      #mpi:: [mvapich2@2.3]
+
+### MPI, Python, MKL
+  mpi:
+    buildable: False
+  intel-oneapi-mpi:
+    externals:
+    - spec: intel-oneapi-mpi@2021.9.0%intel@23.1.0
+      modules:
+      - impi/21.9.0
+      prefix: /opt/intel/oneapi
+  #mvapich2:
+  #  externals:
+  #  - spec: mvapich2@2.3%gcc@9.1.0
+  #    modules:
+  #    - mvapich2-x/2.3
+  #    prefix: /opt/apps/gcc9_1/mvapich2-x/2.3
+  python:
```

Comment:
Are you sure you need lines 24-25?

Reply:
I removed the python definition from here.